docs/quick_start.md

```diff
@@ -0,0 +1,70 @@
+
+# Quick start
+
+## Use sc-machine as a C++ library in your project
+
+### GitHub Releases
+You can download pre-built artifacts from [GitHub Releases](https://github.com/ostis-ai/sc-machine/releases). Extract it to any location, then make it available to CMake by appending folder path to `CMAKE_PREFIX_PATH`:
+
+```cmake
+set(FABULOUS_PROJECT_SC_MACHINE_PATH "/some/path/to/sc-machine-<version>" 
+CACHE PATH "sc-machine installation path"
+)
+
+list(APPEND CMAKE_PREFIX_PATH ${FABULOUS_PROJECT_SC_MACHINE_PATH})
+```
+
+### Conan
+
+You can use Conan to install sc-machine, here's what you'll need to add to `conanfile.txt`:
```

Comment:
Why we don't use conanfile.txt for sc-machine?

Reply:
The conanfile.py is the recipe file of a package, responsible for defining how to build it and consume it.